src/System.IO.FileSystem/tests/Directory/GetSetTimes.cs

```diff
@@ -98,8 +98,8 @@ public void CreationSetsAllTimes()
 
         [OuterLoop]
         [Fact]
-        [ActiveIssue(2369, PlatformID.AnyUnix)]
-        public void DirectoryDoesntExist_ReturnDefaultValues()
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_DirectoryDoesntExist_ReturnDefaultValues()
```

Comment:
I based them heavily off of preexisting tests that were marked as [Outerloop]; after another look, in their current iteration it isn't necessary that they be marked as such.